backend/src/install/cleanup.rs

```diff
@@ -288,5 +293,18 @@ pub async fn uninstall(
         tokio::fs::remove_dir_all(&volumes).await?;
     }
     tx.commit(None).await?;
+    remove_tor_keys(secrets, &entry.manifest.id).await?;
+    Ok(())
+}
+
+#[instrument(skip(secrets))]
+pub async fn remove_tor_keys<Ex>(secrets: &mut Ex, id: &PackageId) -> Result<(), Error>
+where
+    for<'a> &'a mut Ex: Executor<'a, Database = Sqlite>,
+{
+    let id_str = id.as_str();
+    sqlx::query!("DELETE FROM tor WHERE package = ?", id_str)
```

Comment:
Comment: The query looks good. I would have thought you could pass the id as a reference instead of converting to as_str. Then make sure package_id can be used in sqlx queries.